Dockerfile

```diff
@@ -14,5 +14,5 @@ COPY alembic.ini poetry.lock pyproject.toml run.py VERSION* /virtool/
 COPY virtool /virtool/virtool
 COPY assets /virtool/assets
 EXPOSE 9950
-ENTRYPOINT ["python", "run.py"]
+ENTRYPOINT ["adev runserver", "run.py"]
```

Comment:
I would actually leave the dockerfile as is since it provides build instructions for the production ready containers. We only  want hot reloading in dev, so I would instead change the configuration of the `tiltfile` in the `dev` repo.

Reply:
ok; will create a new pr for that repo. I updated the pyproject.toml re. the dev dependency